util.py

```diff
@@ -11,41 +11,34 @@
 log = logging.getLogger(__name__)
 
 
-async def prompt_for_turn(
+async def prompt_menu(
 	bot: commands.Bot,
 	user: discord.User,
-	battlecontext: BattleContext,
+	content: str,
+	title: str,
+	description: str,
+	items: list[tuple[str, str]],
 	use_channel: Optional[discord.TextChannel] = None
-) -> Turn:
-	"""Prompt the given user for a turn.
-
-	:param bot: The discord bot
-	:param user: The discord user
-	:param battlecontext: The :class:`BattleContext` that will be given to the user.
-	:param use_channel: Override the channel that is used to communicate with the user. By default, it will use the user's :class:`discord.DMChannel` to send the prompt. Primarily used for tests.
-	:returns: The turn that the user made.
-	TODO: prompt user for what type of turn
-	"""
+):
+	"""Create menu for the user to choose between several options."""
 
 	if use_channel:
 		channel = use_channel
 	else:
 		if not user.dm_channel:
 			await user.create_dm()
 		channel = user.dm_channel
-	embed = discord.Embed(
-		title=battlecontext.pokemon.Name,
-		description=
-		f"{battlecontext.pokemon.CurrentHP} HP {taggify(type_to_string(battlecontext.pokemon.Type))} {taggify(status_to_string(battlecontext.pokemon.StatusEffects))}"
-	)
-	for i, move in enumerate(battlecontext.pokemon.Moves):
+
+	embed = discord.Embed(title=title, description=description)
+
+	count = 0
```

Comment:
Is this used anywhere?

Reply:
Yeah it is used in the for loop to generate the menu with it's choices. I wasn't able to access the index otherwise to get the letter choices.

Reply:
ah i see